website/docs/getting-started/setup/upgrade-to-business-edition/docker.mdx

```diff
@@ -0,0 +1,129 @@
+---
+description: Follow the steps to upgrade the Appsmith community docker installation to a Business Edition.
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Docker
+
+Follow the steps listed below to **upgrade** your Community Edition (CE) to a Business Edition (EE) installed on docker.
+
+### Step 1: Database backup
```

Comment:
nitpick : Do we need to add 'Step' in the heading. can't it just be **1. Database Backup**